src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -104,6 +105,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddMeetingCommand.COMMAND_WORD:
             return new AddMeetingCommandParser().parse(arguments);
 
+        case EditMeetingCommand.COMMAND_WORD:
+            return new EditMeetingCommandParser().parse(arguments);
```

Comment:
Just add this in the AddressBookParserTest.java
    
    @Test
    public void parseCommand_editc() throws Exception {
        Meeting meeting = new MeetingBuilder().build();
        EditMeetingDescriptor descriptor = new EditMeetingDescriptorBuilder(meeting).build();
        String str = EditMeetingCommand.COMMAND_WORD + " " + INDEX_FIRST_MEETING.getOneBased() + " "
                + MeetingUtil.getEditMeetingDescriptorDetails(descriptor);
        EditMeetingCommand command = (EditMeetingCommand) parser.parseCommand(str);
        assertEquals(new EditMeetingCommand(INDEX_FIRST_PERSON, descriptor), command);
    }